bin/git-node.js

```diff
@@ -27,4 +27,6 @@ Promise.all(commandFiles.map(importCommand)).then((commands) => {
     .epilogue(epilogue)
     .help('help')
     .parse();
+}).catch((err) => {
+  throw err;
```

Comment:
What's the rule for this change?

Reply:
https://github.com/eslint-community/eslint-plugin-promise/blob/HEAD/docs/rules/catch-or-return.md